### ISSUE TITLE
The typed version of racket/fixnum doesn't provide the popcount functions.

### ISSUE THREAD

**Issue by shawnw** [racket/typed-racket#1319]
See title. `fxpopcount`, `fxpopcount32` and `fxpopcount16` need a `require/typed` to be usable.

**Comment by Jobhdez** [comment]
Can I get this assigned? Thanks

**Comment by jbclements** [comment]
Are you asking me to assign this issue to you? I want to make sure I'm not misunderstanding you.

**Comment by Jobhdez** [comment]
> Are you asking me to assign this issue to you? I want to make sure I'm not misunderstanding you.

Yea that's what I'm asking. I'd like to work on this please

**Comment by cocodery** [comment]
Hi, I'm not sure whether the issue is taking on, and the assignee seems to logout. I'm trying this issue and I'm interested in solving this issue.
My solution is just like the typed-racket [date-file](https://github.com/cocodery/typed-racket/blob/master/typed-racket-lib/typed/racket/date.rkt), solves with a new `fixnum` file under the same path.
I have provided all functions in `provide(...)` in racket [fixnum-file](https://github.com/racket/racket/blob/master/racket/collects/racket/fixnum.rkt) except `for/fxvector` `for*/fxvector`.
The two functions are a little tricky for me, do you have any suggestions, or can we slow these two down?

**Comment by samth** [comment]
You probably want to add the types to this file: https://github.com/racket/typed-racket/blob/master/typed-racket-lib/typed-racket/base-env/base-env-numeric.rkt#L1933

**Comment by cocodery** [comment]
Yes, thanks! I have created a PR for this issue about the fxpopcount functions.
